source/engineering/abaqus/multi-node.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Multi-Node Jobs"
+---
+
+Large parallel Abaqus jobs, using more than 28 processors, can be run on across multiple nodes using MPI.
+
+```{tip}
+Multi-node parallelism is best-suited to Abaqus/Explicit.
+Abaqus/Standard (implicit) does not scale well across multiple nodes
+```
+
+## Job Script Template
+
+Here is an example job submission script for a multi-node Abaqus job:
+
+```{code-block} bash

```

Comment:
I'm hesitant to use tabs here because the code block is big and the difference between the two systems is small (one line). Using tabs may imply that there are larger fundamental differences between the submission scripts than simply a different module name. I guess you could quickly flip between tabs to see what is different, but there is now the possibility of the tabs not being kept in sync with future changes.

Reply:
Cheers, I've now changed to `languages/intel/2020-u4`.

I couldn't find this initially with `module spider intel` because the stem has changed - I now notice it is there under `Other possible module matches`. The naming of modules on the clusters is frustratingly inconsistent.